src/Matcher.js

```diff
@@ -125,10 +125,18 @@ export default class Matcher {
       params[name] = value && decodeURIComponent(value);
     });
 
-    return {
-      params,
-      remaining: pathname.slice(match[0].length),
-    };
+    let remaining = pathname.slice(match[0].length);
+    if (this.endsWithDelimiter(match[0])) {
```

Comment:
what does rr do? regardless of what decision you want to take, I do believe that /foo and /foo/ should def be equivalent